clients/client-codecatalyst/src/runtimeConfig.browser.ts

```diff
@@ -30,6 +31,7 @@ export const getRuntimeConfig = (config: CodeCatalystClientConfig) => {
       config?.defaultUserAgentProvider ??
       defaultUserAgent({ serviceId: clientSharedValues.serviceId, clientVersion: packageInfo.version }),
     maxAttempts: config?.maxAttempts ?? DEFAULT_MAX_ATTEMPTS,
+    region: config?.region ?? invalidProvider("Region is missing"),
```

Comment:
this is okay to add to CodeCatalyst, because even now it errors on missing region (due to the config resolver and not the runtimeConfig default).